client/state/selectors/is-fetching-atomic-hosting-wp-version.js

```diff
@@ -0,0 +1,12 @@
+import 'calypso/state/hosting/init';
+
+/**
+ * Whether or not we're currently fetching the WordPress version.
+ *
+ * @param  {Object}  state   Global state tree
+ * @param  {number}  siteId The ID of the site we're querying
+ * @returns {boolean}
+ */
+export function isFetchingAtomicHostingWpVersion( state, siteId ) {
+	return state.atomicHosting?.[ siteId ]?.isFetchingWpVersion;
```

Comment:
Just so the function JSDoc return type is correct.
 
```suggestion
	return state.atomicHosting?.[ siteId ]?.isFetchingWpVersion ?? false;
```